### ISSUE TITLE
titledb.com titles don't work

### ISSUE THREAD

**Issue by suzsuzuki** [Steveice10/FBI#190]
I tried to install 3 titles with the titledb.com option, the app is installed on home screen but can't open or give an error. Download and install the cia from titledb.com works. One example is blarg snes.

I'm in an Old3DS with Luma 6,01 and last FBI release

**Comment by Steveice10** [comment]
What error do they give?

**Comment by suzsuzuki** [comment]
Blarg SNES don't give an error, just black screen after open and I need to keep pressing the power button to power off the 3DS.... BUT I had some tests and it seems pro be a problem from blarg snes, run this app through FBI is successful (but not always). Like this https://www.reddit.com/r/3dshacks/comments/43bzlh/blargsnes_black_screen/

Now the others I tried to install, Orchestrina and MultiUpdater when I try to open the error is "an error has occurred forcing the sw to close. The system will now restart"

But when I tried to install the cia that I downloaded the error is the same, I think the problem is a missing configuration in my 3DS, not a FBI problem.. really sorry.
